app/models/application_qualification.rb

```diff
@@ -144,6 +154,29 @@ def composite_equivalency_details
     details.strip if details.present?
   end
 
+  GCSE_PASS_GRADES = %w[A* A B C A*A* A*A AA AB BB BC CC CD 9 8 7 6 5 4 99 98 88 87 77 76 66 65 55 54 44 43].freeze
```

Comment:
Taken directly from https://trello.com/c/ONl9m9RD/3307-ask-additional-question-if-candidate-does-not-enter-a-gcse-pass-grade